cunumeric/utils.py

```diff
@@ -41,6 +41,14 @@
 ]
 
 
+def _broadcast_shapes(
+    *args: Union[NdShapeLike, Sequence[NdShapeLike]]
+) -> NdShape:
+    # Call 'np.broadcast_shapes' for now.
+    # We will have a new implementation later
```

Comment:
what's the point of this refactoring if you don't have an implementation? bogus changes like this are just a noise for reviewers. please revert the changes unless you mean to fill in this function.

Reply:
5cb1ab8b9754561975ea700ab8ad93398f6608c9